src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js

```diff
@@ -0,0 +1,415 @@
+                that helps you manage your VA prescriptions online.
+              </p>
+              <p>
+                <strong>With this tool, you can:</strong>
+              </p>
+              <ul>
+                <li>Refill your VA prescriptions online</li>
+                <li>View your past and current VA prescriptions</li>
+                <li>
+                  Track the delivery of each prescription mailed within the past
+                  30 days
+                </li>
+                <li>
+                  Get email notifications to let you know when to expect your
+                  prescriptions
+                </li>
+                <li>
+                  Create lists to keep track of all your medicines (including
+                  prescriptions, over-the-counter medicines, herbal remedies,
+                  and supplements)
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        itemScope
+        itemType="http://schema.org/Question"
+        data-template="paragraphs/q_a"
+        data-entity-id={3260}
+        data-analytics-faq-section
+        data-analytics-faq-text="Am I eligible to use this tool?"
+      >
+        <h2 itemProp="name" id="am-i-eligible-to-use-this-tool">
+          Am I eligible to use this tool?
+        </h2>
+        <div
+          data-entity-id={3260}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3261}
+              className="processed-content"
+            >
+              <p>
+                You can use this tool if you meet all of the requirements listed
+                below.
+              </p>
+              <p>
+                <strong>All of these must be true. You:</strong>
+              </p>
+              <ul>
+                <li>
+                  Are enrolled in VA health care, <strong>and</strong>
+                </li>
+                <li>
+                  Are registered as a patient in a VA health facility,{' '}
+                  <strong>and</strong>
+                </li>
+                <li>
+                  Have a refillable prescription from a VA doctor that you’ve
+                  filled at a VA pharmacy and that’s being handled by the VA
+                  Mail Order Pharmacy
+                </li>
+              </ul>
+              <p>
+                <a href="/health-care/how-to-apply/">
+                  Find out how to apply for VA health care
+                </a>
+              </p>
+              <p>
+                <strong>And you must have one of these free accounts:</strong>
+              </p>
+              <ul>
+                <li>
+                  A{' '}
+                  <a href="">
```

Comment:
Missing link

Reply:
@DanielleThierryUSDSVA Any idea where this link should go? 🙂

Reply:
https://www.myhealth.va.gov/mhv-portal-web/upgrading-your-my-healthevet-account-through-in-person-or-online-authentication

Reply:
Do not open in new tab (e.g. no `target="_blank"`) 🙂